### PR TITLE
=pro #21867 update scala-stm dependency for akka-agent, scala 2.12

### LINKED ISSUE #21867: Update dependency of scala-stm to 0.8
So that we can release the agents module also for Scala 2.12

### PR DESCRIPTION
**Pull request by ktoso**
Resolves #21867

### CODE REVIEW

**Comment by ktoso** [comment]
Just works ™

cc @johanandren

**Review by drewhk — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]
LGTM

File: project/Dependencies.scala [hunk 1]
```diff
@@ -19,7 +19,7 @@ object Dependencies {
   val Versions = Seq(
     crossScalaVersions := Seq("2.11.8"), // "2.12.0"
```
Comment (patriknw) [inline project/Dependencies.scala]: is that the last thing needed for using `crossScalaVersions := Seq("2.11.8", "2.12.0")
does our release script already do `+ publishSigned`

**Review by patriknw — APPROVED** [review]
LGTM

File: project/Dependencies.scala (hunk 1 above)
Comment (ktoso) [inline project/Dependencies.scala]: yes, I think so 👍

**Comment by ktoso** [comment]
Tests of akka-agent pass on 2.12.0 with this

**Comment by johanandren** [comment]
Unrelated test failure, I think we should merge this to ease publishing of 2.12 artifact for 2.4.14

**Comment by ktoso** [comment]
I agree.

**Pull request merged**